libraries/AP_Notify/NavioLED_I2C.h

```diff
@@ -17,10 +17,14 @@
 #pragma once
 
 #include "NavioLED.h"
+#include <AP_HAL/I2CDevice.h>
 
 class NavioLED_I2C : public NavioLED
```

Comment:
anyway... since the conversion is already done, we can get it in once the issues pointed above are fixed.